pkg/services/provisioning/dashboards/config_reader.go

```diff
@@ -11,14 +11,45 @@ import (
 	"gopkg.in/yaml.v3"
 
 	"github.com/grafana/grafana/pkg/infra/log"
-	"github.com/grafana/grafana/pkg/services/org"
 	"github.com/grafana/grafana/pkg/services/provisioning/utils"
 )
 
 type configReader struct {
 	path       string
 	log        log.Logger
-	orgService org.Service
+	orgChecker utils.CheckOrgExists
```

Comment:
^^ switched so we just need an "org exists" function rather than the whole service